homeassistant/components/tellduslive/__init__.py

```diff
@@ -70,8 +70,9 @@
 
     client = TelldusLiveClient(hass, entry, session)
     hass.data[DOMAIN] = client
-
-    await client.update()
+    await hass.async_add_executer_job(
```

Comment:
`async_add_hubs` is a coroutine. It should not be scheduled on the thread pool but on the event loop.

Reply:
```suggestion
    await (
```